conftest.py

```diff
@@ -12,3 +14,16 @@ def pytest_configure(config):
     # XXX(dcramer): Kombu throws a warning due to transaction.commit_manually
     # being used
     warnings.filterwarnings("error", "", Warning, r"^(?!(|kombu|raven|sentry))")
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--itunes",
+        action="store_true",
+        help="Run iTunes tests, see tests/sentry/utils/appleconnect/itunes",
+    )
+
+
+def pytest_runtest_setup(item):
+    if item.get_closest_marker("itunes") and not item.config.getoption("--itunes"):
+        pytest.skip("Test requires --itunes")
```

Comment:
why'd you need this? the tests already appear to skip if the creds file is missing

Reply:
It requires user interaction and I wouldn't want to accidentally enable this.  This makes is very specific that this is a weird case.

Reply:
I don't see how one would accidentally enable this